workbench/controllers/RestResponseInstrumenter.php

```diff
@@ -90,4 +95,21 @@ function instrument($s) {
         return preg_replace('@' . $this->pattern . '@', $this->replacement, $s);
     }
 }
-?>
\ No newline at end of file
+
+class RegexUrlInsturmentation implements Insturmentation {
```

Comment:
if this is for URLs, is should all the `urlencoded` business above happen inside here